AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -143,6 +143,8 @@ NS_ASSUME_NONNULL_BEGIN
  * encouraged.
  *
  * @note This method should not be called directly outside of ASDisplayNode; use -measure: or -calculatedLayout instead.
+ *
+ * @warning Overwriting layoutSpecThatFits: in a subclass and providing a layoutSpecBlock block is currently not supported
```

Comment:
Overriding*.  You might say: "Subclasses that implement -layoutSpecThatFits: must not also use .layoutSpecBlock.  Doing so will trigger an exception.  A future version of the framework may support using both, calling them serially, with the .layoutSpecBlock superseding any values set by the method override."